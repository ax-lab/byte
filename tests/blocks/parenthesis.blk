(a)
(a + b + (c + d))

(
	x
)

(
	x1
	(y1
		z1)
)

(
	x2
		(y2
			z2)
)

() + (
	a + (
		b
	)
)
