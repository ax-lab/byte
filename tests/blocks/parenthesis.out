Line(
	P(
		Line(
			a
		)
	)
)
Line(
	P(
		Line(
			a + b + P(
				Line(
					c + d
				)
			)
		)
	)
)
Line(
	P(
		Line(
			x
		)
	)
)
Line(
	P(
		Line(
			x1
		)
		Line(
			P(
				Line(
					y1
					...Line(
						z1
					)
				)
			)
		)
	)
)
Line(
	P(
		Line(
			x2
			...Line(
				P(
					Line(
						y2
						...Line(
							z2
						)
					)
				)
			)
		)
	)
)
Line(
	P() + P(
		Line(
			a + P(
				Line(
					b
				)
			)
		)
	)
)
